tests/Microsoft.Identity.Test.Unit/Microsoft.Identity.Test.Unit.csproj

```diff
@@ -4,8 +4,9 @@
     <TargetFrameworkNetDesktop461>net48</TargetFrameworkNetDesktop461>
     <TargetFrameworkNetCore>netcoreapp3.1</TargetFrameworkNetCore>
     <TargetFrameworkNet6Win>net6.0-windows10.0.17763.0</TargetFrameworkNet6Win>
+    <TargetFrameworkNet7Win>net7.0-windows10.0.18362.0</TargetFrameworkNet7Win>
     
-    <TargetFrameworks>$(TargetFrameworkNetDesktop461);$(TargetFrameworkNetCore);$(TargetFrameworkNet6Win);</TargetFrameworks>
+    <TargetFrameworks>$(TargetFrameworkNetDesktop461);$(TargetFrameworkNetCore);$(TargetFrameworkNet6Win);$(TargetFrameworkNet7Win)</TargetFrameworks>
```

Comment:
Also need to update: https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/main/build/template-run-unit-tests.yaml to run net7 tests.